data/json/items/armor/coats.json

```diff
@@ -1999,8 +1999,8 @@
   {
     "id": "leather_police_jacket",
     "type": "ARMOR",
-    "name": { "str": "leather riding jacket" },
-    "description": "A thick and heavy leather jacket used most commonly by motorcyclists, equestrians and Australian vigilantes.",
+    "name": { "str": "motorcycle jacket" },
+    "description": "A thick and heavy leather jacket used most commonly by motorcyclists, equestrians and Australian vigilantes. The leather is thick enough that it could likely support extra padding.",
```

Comment:
```suggestion
    "description": "A thick and heavy leather jacket used most commonly by motorcyclists, equestrians and Australian vigilantes.  The leather is thick enough that it could likely support extra padding.",
```